File: proyecto_python/proyectofinal2b/problemaAtletismo.py
def obtenerDatosAtletismo():
    print("Ingrese el nombre del participante:")
    nombre = input()

    print("Ingrese la edad del participante:")
    edad = input()
    edad = float(edad)

    print("Ingrese su especialidad (carreras, salto, lanzamiento):")
    especialidad = input()

    print("Ingrese su ciudad de residencia")
    ciudad = input()

    print("Ingrese su record personal en" + especialidad)
    estilo = input()

    cadena = (f"\nLos datos ingresados son: "
              f"\nNombre: {nombre}\nEdad: {edad}\nCiudad: {ciudad}"
              f"\nnivel: {especialidad}\nEspecialiad: {estilo}\n")

    return cadena

File: proyectoJava/proyecto_java/proyecto_final/src/paquete03/ProblemaAtletismo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete03;

import java.util.Scanner;

/**
 *
 * @author utpl
 */
public class ProblemaAtletismo {

    public static String obtenerDatosAtletismo() {

        String cadena;

        Scanner entrada = new Scanner(System.in);
        System.out.println("Ingrese nombre del participante");
        String nombrePersona = entrada.nextLine();

        System.out.println("Ingrese su edad");
        int edad = entrada.nextInt();

        System.out.println("Ingrese su especialidad"
                + " (carreras, salto, lanzamiento)");
        entrada.nextLine();
        String especialidad = entrada.nextLine();

        System.out.println("Ingrese su ciudad de residencia");
        String ciudad = entrada.nextLine();

        System.out.println("Ingrese su record personal en su disciplina");
        String record = entrada.nextLine();

        cadena = String.format("Los datos ingresados para atletismo son: "
                + "\nNombre: %s\nEdad: %d\nCiudad: %s\n"
                + "Record personal: %s\nEspecialidad: %s\n ",
                nombrePersona, edad, ciudad, especialidad, record);

        return cadena;

    }
}
